test/test-lib-nodejs.sh

```diff
@@ -609,5 +609,12 @@ function test_latest_imagestreams() {
   return $result
 }
 
+function cleanup() {
```

Comment:
What about to use https://github.com/sclorg/container-common-scripts/blob/master/test-lib.sh#L154 or this one https://github.com/sclorg/container-common-scripts/blob/master/test-lib.sh#L133?

Reply:
As far as I look both clean containers not images.

Reply:
I seem, then it makes sense